node/src/Utils.h

```diff
@@ -251,6 +252,22 @@ namespace node {
         return size;
     }
 
+    inline bool IsOperator(const Napi::CallbackInfo& info) {
```

Comment:
I don't think this is generic enough, for example unary op only needs to check whether `info.Length() == 0`. How about check them case by case?